### ISSUE TITLE
Future of  module-federation-plugin - angular 17 <

### ISSUE THREAD

**Issue by WeslleyDeSouza** [angular-architects/module-federation-plugin#376]
Hello,

It seems that Angular is transitioning away from Webpack and is leaning towards using Vite.
What might be the future for this library, considering its heavy reliance on Webpack?

**Comment by pamartinez-signify** [comment]
There is [native-federation](https://github.com/angular-architects/module-federation-plugin/tree/main/libs/native-federation) version, which relies on importmap and works with esbuild.
The issue I find with Angular 17 that you get the following message:
The 'browser-esbuild' builder is a compatibility builder which will be removed in a future major version in favor of the 'application' builder.

I did notice the native-federation schematics change the builder defined in angular.json to build with "@angular-devkit/build-angular:browser-esbuild" so I reverted it back to "@angular-devkit/build-angular:application", but the native-federation builder is still referencing the old browser-esbuild internally, so this doesn't really fix the message.

The second and bigger issue (I'm not sure if it's related with this builder) is on `npm run build` it hangs with the message "Building federation artefacts".

For this reason I'ts not really usable until (I hope) version 17 comes out aligned witih Angular 17 builder, and the build task is able to finish.
